lineage/graph.go

```diff
@@ -59,7 +57,7 @@ func (graph InMemoryGraph) Query(cfg QueryCfg) (AdjacencyMap, error) {
 		var err error
 		supergraph, err = graph.buildSubgraphFromRoot(supergraph, cfg.Root)
 		if err != nil {
-			return supergraph, errors.Wrap(err, "error building subgraph")
+			return supergraph, fmt.Errorf("error building subgraph %s", err)
```

Comment:
```suggestion
			return supergraph, fmt.Errorf("error building subgraph: %w", err)
```
If it is error wrapping, we could use `%w`.
see https://go.dev/blog/go1.13-errors

Reply:
Should we replace all error wrapping with `fmt.Errorf` as `errors.wrap` package has been archived.
wdyt ??

Reply:
yes let's do it. I guess that should be no problem, we could do that as part of this story.

Reply:
sounds good